web/src/main/java/org/openmrs/web/filter/initialization/InitializationWizardModel.java

```diff
@@ -81,6 +81,21 @@ public class InitializationWizardModel {
 	
 	public String installMethod = INSTALL_METHOD_SIMPLE;
 	
+	/**
+	 * The type of database being used (mysql or postgresql)
+	 */
+	public String databaseType = "mysql";
```

Comment:
Should this be a constant?

Reply:
I don't this so, the value changes when a user selects PostgreSQL